### ISSUE TITLE
Italian national identity card does not ask whether it's plastic card or folded

### ISSUE THREAD

**Issue by josecanizares** [onfido/onfido-ios-sdk#153]
### What was the expected behaviour?
There's a native modal popup back in version 16.1.0 (possibly even later as well) that, having "ITA" set as the andCountryCode param, would ask if it's a plastic card or folded paper. 

<img width="385" alt="fold" src="https://user-images.githubusercontent.com/57623165/81909018-5037fb80-95ca-11ea-8ef3-fa296c33c4b0.png">

### What happened instead?

With version 18.1.0, it no longer asks this, possibly due to the requirement that configuration is created for each document type.

### Version info:

- Cocoapods (if applicable):
- Onfido SDK (Debug/Release): Release, 18.1.0
- iOS: 10
- Xcode: 11.2.1
- Device/Simulator: iPhone X 13.2.2
- Device/Simulator language: Spanish

### Integration configuration:

It's possible the removal of the andCountryCode parameter causes this.

```var docType: DocumentType;

    switch documentType {
      case "national_id":
        docType = .nationalIdentityCard(config: NationalIdentityConfiguration(country: "ITA"));
      default:
        docType = .drivingLicence(config: DrivingLicenceConfiguration(country: "ITA"));
    }

    
    let onfidoConfig = try! OnfidoConfig.builder()
      .withAppearance(appearance)
      .withSDKToken(token)
      .withDocumentStep(ofType: docType, andCountryCode: countryCode)
      .withFaceStep(ofVariant: .photo(withConfiguration: nil))
      .withCustomLocalization()
      .build()
```

### Steps to reproduce:

Set the country code to Italy and set the document type to national identity card and then start the SDK.

**Comment by anuragajwani** [comment]
Hi @josecanizares,

I am looking into this now. I'll try to reproduce this first.

**Comment by anuragajwani** [comment]
Hi @josecanizares 

This change was introduced in 18.0.0. I have opened a ticket to add the ability for the integrator to preselect document type and country and get the user to select the document format.

Sorry about the inconvenience caused. I will update you once I have more information.
Anurag

**Comment by josecanizares** [comment]
@anuragajwani Great, thanks for looking into it 👍

**Comment by keremgund** [comment]
Hello @josecanizares ,
I'm glad to say that we have just released 18.1.1 version which contains fix for this issue.
Let us know if you think problem still persists.

Kerem

**Comment by josecanizares** [comment]
@keremgund Just tested it and it looks like it's working again. Thanks!

José
